src/styles/stilyagi/motifs.css

```
@@ -363,49 +372,49 @@ section.chapter > .chapter-label {
 }
 
-/* .code-scroll itself is hand-written above the generated marker in
-   syntax.css; the wrapper is a scroller, so it is in the tab order and needs
+/* .code-scroll itself is hand-written, apart from the generated Pygments
+   block in syntax.css; the wrapper is a scroller, so it is in the tab order and needs
    a ring. The ring is drawn inside, over the block's own ink ground, so it is
    paper: 15.21:1 there. */
 .code-scroll:focus-visible {
-  --focus-ring-color: var(--paper);
+  --focus-ring-color: var(--color-paper);
   outline-offset: -3px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)